src/Services/Utils/SQLUpgradeService.php

```diff
@@ -1243,4 +1250,48 @@ private function convertLayoutProperties()
             } // end group
         } // end form
     }
+
+    private function updateLayoutEditOptions($mode, $form_id, $add_option, $values)
+    {
+        $subject = explode(',', str_replace(' ', '', $values));
+
+        $sql = "SELECT `field_id`, `edit_options`, `seq` FROM `layout_options` WHERE `form_id` = ? ";
```

Comment:
would only select the field_id's that are in subject here (this will prevent touching things not related; ie. decrease scope of weird things happening to unrelated entries).

Reply:
actually, i see you are filtering for field_id already below, which will also suffice to protect the unrelated entries (ie. disregard my post above :) )